cliquet/storage/postgresql/__init__.py

```diff
@@ -578,6 +578,12 @@ def _format_conditions(self, filters, id_field, modified_field,
         for i, filtr in enumerate(filters):
             value = filtr.value
 
+            try:
+                value = float(value)
```

Comment:
This autocast is not good enough because it fails with numeric values that aren't like phone numbers.